engine/source/public/core/engine.hpp

```diff
@@ -16,6 +22,11 @@ namespace lumina
         void Initialize();
         void Run();
         void Shutdown();
+
+        FileIO& FileIO() const
+        {
+            return *fileIO;
+        }
```

Comment:
I would prefer to have small functions like this on a single line, but I don't know if clang-format allows it like that. There are multiple things about clang-format I don't like, but  I can't get all the options to work as we want it to.

Reply:
I wanted to do this, but clang automatically reformats it